community/contributing/docs_writing_guidelines.rst

```diff
@@ -274,6 +274,29 @@ element of a list with more than two elements.
 How to write methods and classes
 --------------------------------
 
+Dynamic vs Static typing
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Despite the addition of optional static typing, GDScript remains primarily 
+dynamic language. Unless you are specifily discussing static typing, you
+should only use dynamic typing in your code examples and snipets. 
+This will help reduce confusion to new users by promoting consistent look
+to the code examples they are exposed to. 
+
+**Don't** write using using static typing:
```

Comment:
Assorted grammar/spelling:
- "primarily dynamic" -> "primarily a dynamic"
- "specifily" -> "specifically"
- "snipets" -> "snippets"
- "promoting consistent" -> "promoting a consistent"
- "they are exposed to" -> "to which they are exposed"
- "using using static typing" -> "using static typing"

---
Generally, we will want to make simpler, direct sentences that avoid adverbs and minimize sentence length (at least, in my experience). So, for example:

> GDScript specializes in being concise and easy to learn. Optional static typing conflicts with this vision by making documentation more congested. To improve its accessibility for new users, please use dynamic GDScript code samples where possible. The one exception is topics that explain static typing concepts to users.

Reply:
Thanks. I appreciate your comments it's late now but I will work on this tomorrow and submit all the changes.